src/Arcus.Observability.Telemetry.Core/Extensions/ILoggerMetricExtensions.cs

```diff
@@ -51,5 +66,21 @@ public static void LogMetric(this ILogger logger, string name, double value, Dat
 
             logger.LogWarning(MessageFormats.MetricFormat, new MetricLogEntry(name, value, timestamp, context));
         }
+
+        /// <summary>
+        /// Logs a custom metric
+        /// </summary>
+        /// <param name="logger">The logger to track the metric.</param>
+        /// <param name="name">Name of the metric</param>
+        /// <param name="value">Value of the metric</param>
+        /// <param name="timestamp">Timestamp of the metric</param>
+        /// <param name="context">Context that provides more insights on the event that occurred</param>
+        /// <exception cref="ArgumentNullException">Thrown when the <paramref name="logger"/> is <c>null</c>.</exception>
+        /// <exception cref="ArgumentException">Thrown when the <paramref name="name"/> is blank.</exception>
+        [Obsolete("Use " + nameof(LogCustomMetric) + " instead")]
+        public static void LogMetric(this ILogger logger, string name, double value, DateTimeOffset timestamp, Dictionary<string, object> context = null)
+        {
+            LogCustomMetric(logger, name, value, timestamp, context);
```

Comment:
```suggestion
        {
            Guard.NotNull(logger, nameof(logger), "Requires a logger instance to track telemetry");
            Guard.NotNullOrWhitespace(name, nameof(name), "Requires a non-blank name to track a metric");

            context = context ?? new Dictionary<string, object>();

            LogCustomMetric(logger, name, value, timestamp, context);
```
We should not alter the implementation of obsolete functionality.